frontend/src/components/CalendarQalendar.vue

```diff
@@ -343,6 +344,7 @@ watch(dayBoundary, () => {
     return;
   }
 
+  // TODO: This does update the boundary values, but does NOT update the container height!
   qalendarRef.value.time.DAY_START = dayBoundary.value.start * 100;
   qalendarRef.value.time.DAY_END = dayBoundary.value.end * 100;
```

Comment:
Any ideas how to trigger a UI rerender of Qalendar at this point?

Reply:
We might need to adjust our fork, and then pull in our fork. But I'm not sure 🤔